src/views/GlossaryPage.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+  <DashboardLayout>
+    <DashboardCard>
+      <template #title> Agencies and Organizations </template>
+      <template #content>
+        <div>
+          <p
+            v-for="agency in agencyAcronyms"
+            :key="agency.acronym"
+            class="px-4 py-1 is-size-6"
+          >
+            <strong>{{ agency.acronym }}:</strong>
+            {{ agency.name }}
+          </p>
+        </div>
```

Comment:
```suggestion
        <div class="px-4">
          <GlossaryDefinition
            v-for="agency in agencyAcronyms"
            :key="agency.acronym"
            :term="agency.acronym"
          >
            {{ agency.name }}
          </GlossaryDefinition>
        </div>
```

Then you could use it like this

Reply:
Note: I moved the px-4 out to the div so we style the whole box of contents instead of needing to add that to every single paragraph

Reply:
I tried it both ways and it works in the component itself to indent the whole block of definitions. Will that cause problems in the future? It seemed better to abstract it away and type it once in the component rather than 3 times on the view.

Reply:
Happy to take some best practices recommendations for front end work. Now I see how it highlights the whole block for px-4 on the div in the inspector.